src/backend/edge/edge-message-relay/index.ts

```diff
@@ -14,10 +15,10 @@ export default ({
     edgeShouldRelayAssetStatus: boolean;
     edgeShouldRelayAssetHistory: boolean;
     edgeShouldRelayRules: boolean;
+    topics: string[];
     cacheName?: string;
     collectionName?: string;
 }): ReturnType<typeof relay> => {
-    const topics = [];
```

Comment:
I think we should do something like - 
```
const theTopics = [...topics];

```

And then in all of the below if blocks we would do
```
if (edgeShouldRelayLocation) {
    theTopics.push('<topicName>');
}
```

By creating a new array we won't end up mutating the array that's passed in

Reply:
Done